examples/chef/devices/rootnode_waterleakdetector_0b067acfa3.zap

```diff
@@ -2887,7 +2887,7 @@
               "storageOption": "RAM",
               "singleton": 0,
               "bounded": 0,
-              "defaultValue": "",
+              "defaultValue": "16",
```

Comment:
The actual issue is that Leak Detector should not have the scenes management cluster at all! Please remove it from waterleakdetector.

Reply:
Removed Scenes from waterleak and template .zap